proto/directions.proto

```diff
@@ -0,0 +1,94 @@
+package valhalla;
+
+message Waypoint {
+  optional float lat = 1;
+  optional float lon = 2;
+  optional string type = 3;
+  optional uint32 bearing = 4;
+  optional string name = 5;
+  optional string street = 6;
```

Comment:
is this the street name? is it worth being a StreetName or is that overkill?

Reply:
👍 for changing to `street_name`

Reply:
Currently, this is echoed back from the request. The logic to snap to the supplied street name has not been implemented yet. This is typically just a single string and not the same as a street name list that exists for the maneuver or step.

Reply:
Ahh, I see

Reply:
If supplied, the `street` is used for the destination instruction - for example: `123 Main Street is on the right`